instrumentation/src/AndroidManifest.xml

```diff
@@ -0,0 +1,12 @@
+<?xmxl version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="org.oppia.android.app.instrumentation">
+  <uses-sdk
+    android:minSdkVersion="19"
+    android:targetSdkVersion="28" />
+  <instrumentation
+    android:targetPackage="org.oppia.android"
+    android:name="androidx.test.runner.AndroidJUnitRunner" />
+  <application
+    android:label="OppiaTest" />
```

Comment:
This can fit on one line, so it should be.

Reply:
Done